engine/scripting/private/utility/foreign_bindings.cpp

```diff
@@ -0,0 +1,48 @@
+#include "utility/foreign_bindings.hpp"
+#include "log.hpp"
+#include "utility/script_file_loading.hpp"
+
+ForeignBindings::ForeignBindings(const std::string& engineModule)
+{
+    _engineModule = ScriptFileLoading::CanonicalizePath(engineModule);
```

Comment:
can be done in initializer

Reply:
For sure... but I think I will rework this class a bit since its a bit convoluted